api/api.go

```diff
@@ -3,6 +3,7 @@ package api
 import (
```

Comment:
Because of the way we're currently subpackaging `api` with a vendor folder, you'll need to run `make sync` to ensure these `api` package changes get picked up everywhere.

Reply:
I will run on my next push thanks for pointing that out

Reply:
Done